psydac/api/tests/test_api_2d_system_mapping_multipatch.py

```diff
@@ -0,0 +1,203 @@
+# -*- coding: UTF-8 -*-
+
+import pytest
+import numpy as np
+from mpi4py import MPI
+from sympy  import pi, sin, cos, Tuple, Matrix
```

Comment:
Consistently with the latest changes in `devel`, please rename this file as `test_2d_multipatch_mapping_maxwell.py`

Reply:
done in [9b8a549](https://github.com/pyccel/psydac/pull/160/commits/9b8a549c956b3c768890aa18b15c3979fa48e517)